lua/header.lua

```diff
@@ -160,7 +167,40 @@ local function add_license_header(opts)
         local commented_headers = comment_headers(license_table, comments)
         vim.api.nvim_buf_set_lines(buffer, 0, 0, false, commented_headers)
     else
-        print("Unsupported file type:", file_extension)
+        vim.notify_once("unsupported file type for adding header", vim.log.levels.ERROR)
+    end
+end
+
+function header.update_date_modified()
+    local buffer = vim.api.nvim_get_current_buf()
+    local file_extension = vim.fn.expand("%:e")
+    local comments = filetype_table[file_extension]() -- Retrieve the comment style for the current file type
+    local lines = vim.api.nvim_buf_get_lines(buffer, 0, 8, false)
+
+    -- First, determine if there is a header by looking for the "Date created" line
```

Comment:
Do not assume `Date Created` is present as it is also optional option, which can be **disabled** by the customers. Use the `is_header_line` method instead.